services/web/server/src/simcore_service_webserver/resource_manager/user_sessions.py

```diff
@@ -117,9 +117,9 @@ async def remove_socket_id(self) -> None:
         )
 
         await self._registry.remove_resource(self._resource_key(), _SOCKET_ID_FIELDNAME)
-        await self._registry.set_key_alive(
-            self._resource_key(), _get_service_deletion_timeout(self.app)
-        )
+        # when the tab is closed the alive key is also removed immediately,
+        # there is no reason to keep it active
+        await self._registry.set_key_alive(self._resource_key(), 1)
```

Comment:
the key alive has a TTL.
I checked that this function _get_service_deletion_timeout (which is badly named) returns the default TTL.
Did you actually check what happens when you close the tab now? it will instantly remove the service when the GC runs.

This kind of disconnection happen a lot in a train for example... so I think here you might be messing around with it. Did you test that use-case?